python/lsst/ip/isr/fringe.py

```diff
@@ -108,14 +121,27 @@ def run(self, exposure, fringes, seed=None):
         Primary method of FringeTask.  Fringes are only subtracted if the
         science exposure has a filter listed in the configuration.
 
-        @param exposure    Science exposure from which to remove fringes
-        @param fringes     Exposure or list of Exposures
-        @param seed        32-bit unsigned integer for random number generator
+        Parameters
+        ----------
+        exposure : `lsst.afw.image.Exposure`
+            Science exposure from which to remove fringes.
+        fringes : `lsst.afw.image.Exposure` or `list` thereof
```

Comment:
Does it have to be a list or just an iterable? I didn't see iterable anywhere in the diffs.

Reply:
It can probably be any iterable.  This value comes from readFringes, which in turn pulls it from the butler, and I've naively assumed that that would return a list of entries if multiple items were found.